DataFormats/L1THGCal/interface/HGCalTriggerCell.h

```diff
@@ -55,6 +61,9 @@ namespace l1t
             
             double mipPt_;
 
+            uint32_t uncompressedCharge_;
+            uint32_t compressedCharge_;
```

Comment:
A comment has been made by Gianluca that it would be better to initialize these class members. For instance with in-class initialization
```cpp
double mipPt_{0.};
uint32_t uncompressedCharge_{0};
uint32_t compressedCharge_{0};
```
Could I ask you also to update the `mipPt` line, even if you didn't write it?

Reply:
I wonder if there is any convention within CMS when it comes to member initialization? I notice that in most places within the CMSSW code, most member got initialized in the constructor.

Reply:
Done this too!